packages/frontend/src/components/create-campaign/form/rewards-step/index.tsx

```diff
@@ -179,6 +181,11 @@ export function RewardsStep({
             : [reward];
 
         setOpen(false);
+        setFeedbackVisible(true);
+        setTimeout(() => {
```

Comment:
Wondering if we should move the timeout to a `useEffect` so we can also clear it on unmount, for example:

```tsx
useEffect(() => {
    if (!feedback) return;

    const timeout = setTimeout(() => {
        setFeedbackVisible(false);
    }, 1000);

    return () => clearTimeout(timeout);
}, [feedback]);
```
Wdyt?

Reply:
Yes, I'll do it.

Reply:
Let's have `1500` for the feedback here, to be consistent with the address copy.